packages/caver-wallet/src/index.js

```diff
@@ -16,13 +16,307 @@
     along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
 */
 
+const _ = require('lodash')
+const AccountLib = require('eth-lib/lib/account')
 const Keyring = require('./keyring/keyring')
+const TransactionHasher = require('../../caver-transaction/src/transactionHasher/transactionHasher')
+const { KEY_ROLE } = require('./keyring/keyringHelper')
+const utils = require('../../caver-utils')
 
-// TODO: This will be implemented soon.
+/**
+ * representing a Keyring container which manage keyrings.
+ * @class
+ */
 class KeyringContainer {
-    constructor() {
+    /**
+     * creates a keyringContainer.
+     * @param {Array.<Keyring>} keyrings - The keyrings to manage in keyringContainer.
+     */
+    constructor(keyrings) {
+        keyrings = keyrings || []
+        this._length = 0
+        this._addressKeyringMap = new Map()
+
+        // add keyrings to keyringContainer
+        for (const keyring of keyrings) {
+            this.add(keyring)
+        }
+
         this.keyring = Keyring
     }
+
+    /**
+     * @type {number}
+     */
+    get length() {
+        return this._length
+    }
+
+    /**
+     * generates keyrings in the keyringContainer with randomly generated key pairs.
+     *
+     * @param {number} numberOfKeyrings The number of accounts to create.
+     * @param {string} [entropy] A random string to increase entropy. If undefined, a random string will be generated using randomHex.
+     * @return {KeyringContainer}
+     */
+    generate(numberOfKeyrings, entropy) {
+        for (let i = 0; i < numberOfKeyrings; ++i) {
+            this.add(Keyring.generate(entropy))
+        }
+        return this
+    }
+
+    /**
+     * generates a private key string
+     *
+     * `caver.wallet.generatePrivateKey()`
+     *
+     * @param {string} entropy A random string to increase entropy.
+     * @return {string}
+     */
+    // eslint-disable-next-line class-methods-use-this
+    generatePrivateKey(entropy) {
```

Comment:
Do we need this function in KeyringContainer?

Reply:
I create a Wallet class that wraps KeyringContainer inside the caver-wallet package, and the generatePrivateKey function is implemented inside the Wallet.

Reply:
@jimni1222 Wallet is another class? Then, what about adding this function to the Wallet class, not in KeyringContainer?

Reply:
@jimni1222 Oh, it is as what I said. Please ignore this comment. :)